### ISSUE TITLE
0xdec41ab8a95cd64e9952addea8ee8afb11c294b9 to analyse

### ISSUE THREAD

**Issue by branciard** [iExecBlockchainComputing/public-pool-scripts#43]
check workeroder : 0x738574704e8879a570e1a05c049254b9d7ae5aeb

**Comment by branciard** [comment]
Contribution ok but order not finalized. Manual reward TODO.

```

iexec work show 0x738574704e8879a570e1a05c049254b9d7ae5aeb --chain mainnet
ℹ iExec SDK update available 2.2.39 →  2.3.1, Run "npm -g i iexec" to update

ℹ using chain [mainnet]
✔ work 0x738574704e8879a570e1a05c049254b9d7ae5aeb status is ACTIVE, details:
m_workerpool:          0x0ed5e70874487bf21ee65343fb6c5fa13382cdb6
m_params:              {"cmdline":"monte-carlo normal"}
m_requester:           0x8ec966ff9752c5de46105155ef1e9a3a38542358
m_app:                 0xcb0bba8cf239e167af92a51c4212adbf9b479f6a
m_dataset:             0x0000000000000000000000000000000000000000
m_emitcost:            0
m_uri:
m_stdout:
m_resultCallbackProof: 0x0000000000000000000000000000000000000000000000000000000000000000
m_iexecHubAddress:     0x0d5ef019ca4c5cc413ee892ced89d7107c5f424d
m_callback:            0x0000000000000000000000000000000000000000
m_status:              1
m_marketorderIdx:      368
m_stderr:
m_beneficiary:         0x0000000000000000000000000000000000000000
m_statusName:          ACTIVE

ℹ if work is not "COMPLETED" after Wed Dec 19 2018 22:46:31 GMT+0100 (CET) you can claim the work to get a full refund using "iexec work claim"

```

**Comment by branciard** [comment]
@Vince427 23 manual reward done for worker 0xdec41ab8a95cd64e9952addea8ee8afb11c294b9 see tx    https://etherscan.io/tx/0x9b1af1841aeabbabd9beb669275bb2cede1139e88b647e8a538934afd40245e8
